cfgov/v1/models/home_page.py

```diff
@@ -54,4 +69,42 @@ def get_category_name(self, category_icon_name):
 
     def get_context(self, request):
         context = super(HomePage, self).get_context(request)
+        context['latest_updates'] = self.get_latest_updates(request)
         return context
+
+    def get_latest_updates(self, request):
+        # TODO: There should be a way to express this as part of the query
+        # rather than evaluating it in Python.
```

Comment:
Is it worth putting a few more minutes into trying to figure this out now? I predict a long life for this TODO if we merge it as-is.

Reply:
A few minutes more than the several hours I already put in to that very thing? 😉 

It's mostly a Kobayashi-TODO: there to admonish anyone  (including future-me) who looks at that file but doesn't try. I do not believe it's possible right now with the Django ORM, but I'm happy to be wrong about that if anyone else wants to try.

Reply:
I should note, [`difference()`](https://docs.djangoproject.com/en/1.11/ref/models/querysets/#difference) would be a lovely solution here if it didn't mean the resulting queryset [could only be sliced or value'd](https://docs.djangoproject.com/en/1.11/ref/models/querysets/#union).

Reply:
What do you need to do with this queryset that isn't a slice or value? Could you put the `difference` at the end, after the `order_by`?

Reply:
When rendering these it's nice to have access to the objects [for the category handling](https://github.com/cfpb/cfgov-refresh/pull/4894/files#diff-0763362c3528149de7f93e20ad534473R201) and [`get_url()`](https://github.com/cfpb/cfgov-refresh/pull/4894/files#diff-0763362c3528149de7f93e20ad534473R214). We could do both of those without... but it seems less than ideal.